### PR TITLE
Recs codebook

### PR DESCRIPTION
**Pull request by ivelasq**
I still need to add some info on the weighting variables.

### CODE REVIEW

**Comment by ivelasq** [comment]
Hmm, I reran the ANES codebook and it created a conflict. Need to figure out why that happened...

File: Codebook/RECS 2015 Codebook.Rmd [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+recs %>%
+   count(HousingUnitType) %>%
+   mutate(`Unweighted Freq` = round(n / sum(n), 3)) %>%
+   kbl() %>%
+   kable_minimal()
+```
+
+Variables used: TYPEHUQ
+
+### YearMade
```
Comment (szimmer) [inline Codebook/RECS 2015 Codebook.Rmd:101]: YearMadeRange

File: Codebook/RECS 2015 Codebook.Rmd [hunk 2]
```diff
@@ -0,0 +1,575 @@
...
+```{r variable-SpaceHeatingUsed}
+recs %>%
+   count(SpaceHeatingUsed) %>%
+   mutate(`Unweighted Freq` = round(n / sum(n), 3)) %>%
+   kbl() %>%
+   kable_minimal()
+```
+
+Variables used: HEATHOME
+
```
Comment (szimmer) [inline Codebook/RECS 2015 Codebook.Rmd]: Section should be SpaceHeatingUsed

**Comment by szimmer** [comment]
Can you update README file to include Codebook?

**Pull request merged**